src/chips/hotel/src/interrupts.rs

```diff
@@ -1,207 +1,209 @@
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_CPU0_I_IF_BUS_ERR_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_CPU0_I_IF_UPDATE_WATCHDOG_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_CPU0_S_IF_BUS_ERR_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_CPU0_S_IF_UPDATE_WATCHDOG_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_DDMA0_IF_BUS_ERR_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_DDMA0_IF_UPDATE_WATCHDOG_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_DSPS0_IF_BUS_ERR_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_DSPS0_IF_UPDATE_WATCHDOG_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_DUSB0_IF_BUS_ERR_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_DBCTRL_DUSB0_IF_UPDATE_WATCHDOG_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_FUSE0_FUSE_DEFAULTS_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_ALERT_GROUPA_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_ALERT_GROUPB_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_ALERT_GROUPC_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_DIFF_FAIL_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_FW0_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_FW1_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_FW2_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_FW3_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_HEARTBEAT_FAIL_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_PROC_OPCODE_HASH_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_GLOBALSEC_SRAM_PARITY_SCRUB_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_KEYMGR0_AES_EXEC_CTR_MAX_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_KEYMGR0_AES_HKEY_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_KEYMGR0_CERT_LOOKUP_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_KEYMGR0_FLASH_ENTRY_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_KEYMGR0_PW_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_KEYMGR0_SHA_EXEC_CTR_MAX_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_KEYMGR0_SHA_FAULT_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_KEYMGR0_SHA_HKEY_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_PMU_BATTERY_MON_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_PMU_PMU_WDOG_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_RTC0_RTC_DEAD_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_TEMP0_MAX_TEMP_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_TEMP0_MAX_TEMP_DIFF_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_TEMP0_MIN_TEMP_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_TRNG0_OUT_OF_SPEC_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_TRNG0_TIMEOUT_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_VOLT0_VOLT_ERR_ALERT_INT
+      unhandled_interrupt, // GLOBALSEC_XO0_JITTERY_TRIM_DIS_ALERT_INT
+      ::gpio::gpio0_0_handler, // GPIO0_GPIO0INT
+      ::gpio::gpio0_1_handler, // GPIO0_GPIO1INT
```

Comment:
I was lazy and only added two pin interrupts here. Sue me.

As soon as this gets merged I'm going to replace this whole array with a single unified handler that just does all the boiler plate, so we won't need explicit interrupt handlers in any of the controllers (they all do the same thing anyway), and the only place you'll have to care is in `chip.rs`. @yuriks originally suggested it and we should just do that.